stubs/iterable.stub

```diff
@@ -157,15 +157,12 @@ class ArrayIterator implements SeekableIterator, ArrayAccess, Countable
  */
 class RecursiveIteratorIterator
 {
-
 	/**
 	 * @param T $iterator
-	 * @param int $mode
-	 * @param int $flags
```

Comment:
it seems as soon as I leave these phpdocs here, the signature from `resources/functionMap_bleedingEdge.php´ does not work, as it starts failling

```
There was 1 failure:

1) PHPStan\Rules\Classes\InstantiationRuleTest::testBug10324
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
-'23: Parameter #3 $flags of class RecursiveIteratorIterator constructor expects 0|16, 2 given.
+'
 '

/Users/staabm/workspace/phpstan-src/src/Testing/RuleTestCase.php:159
/Users/staabm/workspace/phpstan-src/tests/PHPStan/Rules/Classes/InstantiationRuleTest.php:466

```